### DISCUSSION TITLE
sell Heart of the Oak issue

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#326] [Q&A]
Hi. My Necromancer made Heart of the Oak.
but he sold to NPC.
I found message from console history.

Runewords: Made runword: Heart of the Oak Knout
...
LowGold sell Knout

this is not first time selling runwords issue.
please check this.

**Comment by isid0re** [comment]
hello, 
have you updated to the latest repo? The false negative on autoequipcheck has been fixed. That was the cause of selling/dropping runewords/items it was unable to equip. 

please let me know, thanks.
-isi

**Reply by mirakhany** [reply]
Hi. Yes. I always using latest repo when I posting.
I tested 2021.08.07 and 2021.08.09 version again.
My necromancer(Leveling build) made and sold Heart of the Oak immediately in both version.
it's same console message.
LowGold sell Flail
he has 1463169 gold in stash.
dex is 32(with gears, charms). not enough to equip flail(need 35dex)

**Reply by isid0re** [reply]
Thanks for posting again. I will be investigate it and work out a cause and solution. Thanks
